docker-compose.yml

```diff
@@ -33,6 +33,8 @@ services:
     build: frontend
     networks:
       - apollo-network
+    environment: 
```

Comment:
@mariodanilosilva, we do not need this parameter set in the docker compose. It is only add it to main.js file on vuejs project.